src/perfectoid_space.lean

```diff
@@ -5,25 +5,29 @@ import Tate_ring
 import power_bounded
 import for_mathlib.topological_groups -- for the predicate is_complete_hausdorff
 
-universe u
-
 -- notation for the power bounded subring
 local postfix `ᵒ` : 66 := power_bounded_subring
 
 open nat.Prime power_bounded_subring topological_space
 variable [nat.Prime] -- fix a prime p
 
 /-- A perfectoid ring, following Fontaine Sem Bourb-/
```

Comment:
Given every help Bourbaki gave us (in books and by email), I think we can write his name in full

Reply:
Done